tests/h/views/activity_test.py

```diff
@@ -1095,6 +1111,7 @@ def group_annotation_count(self, pubid):
 def group(factories):
     group = factories.Group()
     group.members.extend([factories.User(), factories.User()])
+    group.authority = "biopub.hypothes.is"
```

Comment:
This should probably be a separate test fixture which is used only in tests that need it.